src/modules/admin/admin.routes.js

```diff
@@ -0,0 +1,8 @@
+const {getAdmins,createAdmin,loginAdmin,updateAdmin} = require('./admin.controller')
+
+module.exports = (app) => {
+    app.get('/admins', getAdmins);
+    app.post('/admins', createAdmin);
```

Comment:
Vai , please update the name of urls something  like '/admin/signup' , I think It will me more meaningful and easy to understand.

Reply:
solve this one bhai

Reply:
ok vai .. cool